llvm/test/Instrumentation/AddressSanitizer/kcfi.ll

```diff
@@ -1,6 +1,9 @@
 ;; Test that we emit kcfi_type metadata for asan.module_ctor with KCFI.
 
-; RUN: opt < %s -passes=asan -S | FileCheck %s
+; Added -opaque-pointers.
```

Comment:
I hope we switch to opaque pointers soon and we need to revert all changes like this.
Unfortunately, it's not clear how to find them. Can we list all the tests we need to update in one place (e.g. GitHub issue)?
These tests won't break after we switch to opaque pointers, so it's easy to forget about them.

Reply:
We had the similar issue with switching to new pass manager, some changes to revert were found several months later after the actual switch. 
But I believe all these test changes can be actually found by the commit message and/or by the respective comments which Maria has already left in the code.

Reply:
Not all of these test changes were made with the comments, unfortunately. I can create a GitHub issue to track changed tests, but I'm afraid I don't have enough time to find names of previously changed tests to put it there.

Reply:
> I can create a GitHub issue to track changed tests, but I'm afraid I don't have enough time to find names of previously changed tests to put it there.

:(. Okay, but let's make sure we don't lose the track from now on. We need some simple way to find the tests we need to update after the switch. Instead of explicitly listing them in the issue, there might be some unique string, which we can easily find with the tools like grep and instructions in the issue how to find these places to update.

+@jcranmer-intel.